src/lua/compat.lua

```diff
@@ -0,0 +1,333 @@
+-- compat.lua -- internal module intended to solve compatibility problems in
+-- different parts of Tarantool. Introduced in gh-7000, see also gh-6912.
+
+local NEW = true
+local OLD = false
+
+-- luacheck: no unused
+local internal = require('internal.compat')
+
+local options_format = {
+    default    = 'string',
+    brief      = 'string',
+    obsolete   = 'string/nil',
+    run_action = 'boolean/nil',
+    action     = 'function/nil',
+}
+
+-- Contains options descriptions in following format:
+-- * default  (string)
+-- * brief    (string)
+-- * obsolete ('X.Y' version or nil)
```

Comment:
I guess X.Y is forgotten here and it should be 'string or nil'.

Reply:
Fixed.